pandas/core/frame.py

```diff
@@ -10641,7 +10641,11 @@ def apply(
             raise ValueError(f"Unknown engine {engine}")
 
     def map(
-        self, func: PythonFuncType, na_action: Literal["ignore"] | None = None, **kwargs
+        self,
+        func: PythonFuncType,
+        skipna: bool = False,
+        na_action: Literal["ignore"] | None | lib.NoDefault = lib.no_default,
```

Comment:
I think these should be switched to not break users passing arguments by position